### ISSUE TITLE
Error: cannot access member `statusCode' of deleted QObject

### ISSUE THREAD

**Issue by agurha** [sheebz/phantom-proxy#9]
Hi ,

Im getting error on ubuntu 

Error: cannot access member `statusCode' of deleted QObject",[{"file":"/home/ubuntu/www/node_modules/phantom-proxy/lib/server/webpage.js","line":117,"function":""}]]

Would you be able to find out why am i getting this ?. The same code works fine on my mac. 

Many thanks,
A

**Comment by sheebz** [comment]
Hi, I suspect that phantom/Webkit is doing some cleanup and you are accessing an object that has been deleted.  Is it possible that you are trying to access a page after proxy.end has fired?  There could also be a bug here, I haven't been able to get all the unit tests to pass on ubuntu.

**Comment by estliberitas** [comment]
I have the same issue. And it's not caused by dead or end()-ed proxy object. It's caused by callback which can be called twice. I had this error thrown here (`includeJs()`):
https://github.com/sheebz/phantom-proxy/blob/master/lib/server/webpage.js#L102

I did simple check:
http://gyazo.com/2373fbf2404df9192f69884ab56ad79e
And what I saw in console:
http://gyazo.com/c5da0cd246d20c8becaeedaea8241a27

So includeJs callback was called twice. But in my code I do only one `proxy.page.includeJs(url, callback)`

Here is this problem described:
http://stackoverflow.com/questions/11597990/phantomjs-ensuring-that-the-response-object-stays-alive-in-server-listen

So sometimes callback is called twice and it's better to check this situation. Or maybe some callback is saved as listener and not detached when for example includeJs() has acted? I'm only guessing... For example I use same page many times, so something may be not cleared.

Anyways, if I find solution, I'll do pull-request because this bug is really annoying.
